app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+
+  validates :title, presence: true, length: { minimun: 5 }, too_short: 'Get serious!!!'
+  validates :body, presence: true, length: { minimun: 5 }, too_short: 'Come on!!! This is toooooo short'
```

Comment:
I think you meant `minimum1` instead of `minimum`

Reply:
Issues raised have been fixed